pkgs/desktops/gnome/extensions/default.nix

```diff
@@ -56,16 +56,17 @@ let
 
 in rec {
   # Remember to import all these in all-packages.nix
-  gnome38Extensions = mapUuidNames (produceExtensionsList "38");
-  gnome40Extensions = mapUuidNames (produceExtensionsList "40");
-  gnome41Extensions = mapUuidNames (produceExtensionsList "41");
-  gnome42Extensions = mapUuidNames (produceExtensionsList "42");
-  gnome43Extensions = mapUuidNames (produceExtensionsList "43");
-  gnome44Extensions = mapUuidNames (produceExtensionsList "44");
-  gnome45Extensions = mapUuidNames (produceExtensionsList "45");
+  gnome38Extensions = mapUuidNames (produceExtensionsList "38") // { recurseForRelease = false; };
```

Comment:
I find this diff rather unsatisfying. Would it be possible to move the `//` up into `all-packages.nix` instead?

Reply:
How about 1d5a3536372ef2eec1725adf275d66f12a282f2d?